### PR TITLE
Shrink deployment yaml to allow for deploying community/commercial docs into separate projects

### PR DESCRIPTION
**Pull request by wgordon17**

### CODE REVIEW

**Comment by wgordon17** [comment]
@vikram-redhat

File: contributing_to_docs/tools_and_setup.adoc [hunk 1]
```diff
@@ -116,3 +116,54 @@ edit existing content or create assemblies and modules.
 some basic guidelines to keep things consistent across our content.
 * link:create_or_edit_content.adoc[Create a local working branch] on your
 workstation to edit existing content or create content.
+
+=== How to deploy to OpenShift
```
Comment (vikram-redhat) [inline contributing_to_docs/tools_and_setup.adoc]: I would change this to: How to deploy to your OpenShift cluster for testing

File: contributing_to_docs/tools_and_setup.adoc [hunk 2]
```diff
@@ -116,3 +116,54 @@ edit existing content or create assemblies and modules.
...
+You can deploy to OpenShift for development. This process will use your github repo to launch the website,
```
Comment (vikram-redhat) [inline contributing_to_docs/tools_and_setup.adoc]: Same thing, clarify that this is on your own instance of OpenShift cluster. Does this matter if the cluster is 3.x or 4.x?

**Comment by vikram-redhat** [comment]
@wgordon17 couple of comments, but otherwise LGTM

File: contributing_to_docs/tools_and_setup.adoc (hunk 2 above)
Comment (wgordon17) [reply contributing_to_docs/tools_and_setup.adoc]: No difference on v3 vs v4 deployment

**Comment by vikram-redhat** [comment]
@wgordon17 ok to merge?

**Comment by wgordon17** [comment]
Yessir 👍

**Pull request merged**